src/modules/common/components/amountField/index.js

```diff
@@ -76,7 +77,7 @@ const AmountField = ({
   useEffect(() => {
     if (isMaximum) {
       const value = formatAmountBasedOnLocale({
-        value: fromRawLsk(maxAmount.value),
+        value: convertFromBaseDenom(maxAmount.value, token),
```

Comment:
There's still 19 more files where we use `fromRawLsk ` and 3 more files where we use `toRawLsk`. Should we replace these too?

Reply:
hmm i guess those are in the test files okay I would take a look